Heap/1962RemoveStonestoMinimizetheTotal.py

```
@@ -0,0 +1,16 @@
+import heapq
+import math
+class Solution:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)